test/valgrind/valgrindTestSuite.sh

```diff
@@ -259,64 +241,6 @@ function add()
 
 
 
-# -----------------------------------------------------------------------------
-#
-# Start broker
-#
-function brokerStart()
-{
-    # Starting contextBroker in valgrind with a clean database
-    bsResult=0
-    dbReset "$1"
-    killall contextBroker 2> /dev/null
-    echo 'db.dropDatabase()' | mongo valgrindtest --quiet > /dev/null
-    valgrind --memcheck:leak-check=full --show-reachable=yes --trace-children=yes contextBroker -port ${CB_TEST_PORT} -db leaktest -harakiri -t0-255 > ${NAME}.out 2>&1 &
-    valgrindPid=$!
-
-    # Awaiting valgrind to start contextBroker (sleep a maximum of 10 seconds)
-    typeset -i loopNo
-    typeset -i loops
-    loopNo=0
-    loops=100
-
-    while [ $loopNo -lt $loops ]
-    do
-      nc -w 2 localhost ${CB_TEST_PORT} &>/dev/null </dev/null
-      if [ "$?" == "0" ]
-      then
-        vMsg The orion context broker has started, listening on port $CB_TEST_PORT
-        sleep 1
-        break;
-      fi
-      vMsg Awaiting valgrind to fully start the orion context broker '('$loopNo')' ...
-      sleep .2
-      loopNo=$loopNo+1
-    done
-
-    if [ $loops == 100 ]
-    then
-        bsResult=tmo
-    fi
-}
-
-
-
-# -----------------------------------------------------------------------------
-#
-# Stop broker
-#
-function brokerStop()
```

Comment:
Unused function (I guess it comes from the time .vtest files were used but they were removed time ago).

NTC (just informative)